schemas/cs3/tbl/t_item/item.ksy

```diff
@@ -0,0 +1,109 @@
+meta:
+  id: item
+  endian: le
+  encoding: UTF-8
+  imports:
+    - /components/cs3/character
+    - /components/cs3/item_type
+    - /components/cs3/effect
+    - /components/common/element
+seq:
+  - id: item_id
+    type: u2
+  - id: character_restriction
+    type: character
+  - id: flags
+    type: strz
+  - id: item_type
+    type: item_type
+  - id: item_subtype
+    type: u2 # TODO: make an enum for this?
+
+  - id: element
+    type: u1 # can't use element enum here because it's a u2
```

Comment:
Hmm this is a shame, I'd rather not having duplicate enums so not sure how to deal with this yet

Reply:
An easy workaround would be to make the enum a u1. Wherever it's encoded as u2, we encode it as that enum followed by a null byte.